lib/index.js

```diff
@@ -131,7 +131,15 @@ MongoStorage = (function() {
         return callback(null, res);
       })["catch"](callback);
     });
-    return req.pipe(busboy);
+    //return req.pipe(busboy);
+    
+    // There may be a requirement to read the req stream before calling this function. e.g. to do virus scanning in temporary location.
+    // The file stream can again be attached to req.filedata and passed to this function in that case.
```

Comment:
Yes @jdrouet. We store in a folder on file system and then execute a virus scan script on it. The folder is quarantine. 

Once the virus scan command returns okay, we upload it in database. We dont want to upload in database until it passes virus scanning tests.